packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -463,7 +473,7 @@ export default ({ api }: { api: APIType }) => {
       yield put(
         A.setMarketplaceData({
           // @ts-ignore
-          collection: IS_TESTNET ? { ...res, collection_data: { ...res } } : res
+          collection: res
```

Comment:
I think this will break testnet but checking

Reply:
Yeah think we need this

Reply:
Not sure why the BE returns different shapes for this call, I think for testnet we reroute directly to openseas api via service-explorer-gateway and for mainnet hit our internal db

Reply:
From pulling from our api locally on testnet, had to change this so collection_data would come through